pages.yml

```diff
@@ -0,0 +1,8 @@
+---
+- accessibility
+- agile
+- testing-cookbook
+- before-you-ship
+- guides
+- from: first-thing
+  to: other-thing
```

Comment:
Er this is the real, "production" `pages.yml` right?  Should the `first-thing`/`other-thing` entry actually be in here?